### ISSUE TITLE
Discussion: better terminology for “live reload”

### ISSUE THREAD

**Issue by chasenlehara** [stealjs/live-reload#30]
@justinbmeyer brought up [here](https://github.com/bitovi/presentations/issues/1) that our use of “live reload” isn’t quite right, since what we really do is hot module swapping and not just reload the entire page.

I’m creating this issue so we can consider using better terminology in places like this: https://github.com/stealjs/steal-tools/blob/c6f735639ddffe17fed8d35ea591e217c8ab87fa/lib/stream/live.js#L64

**Comment by matthewp** [comment]
Yeah, sure, what should we call it in that case.  steal-hot-swap?  "The hot swap server is not running"?
